src/main/java/org/togetherjava/discord/server/CommandHandler.java

```diff
@@ -28,13 +33,14 @@
   private static final Pattern CODE_BLOCK_EXTRACTOR_PATTERN = Pattern
       .compile("```(java)?\\s*([\\w\\W]+)```");
 
-  private JShellSessionManager jShellSessionManager;
+  private final JShellSessionManager jShellSessionManager;
   private final String botPrefix;
-  private RendererManager rendererManager;
-  private boolean autoDeleteMessages;
-  private Duration autoDeleteMessageDuration;
-  private InputSanitizerManager inputSanitizerManager;
+  private final RendererManager rendererManager;
+  private final boolean autoDeleteMessages;
+  private final Duration autoDeleteMessageDuration;
+  private final InputSanitizerManager inputSanitizerManager;
   private final int maxContextDisplayAmount;
+  private final Map<Long, Message> answeredMessages = new HashMap<>();
```

Comment:
implement housekeeping of that map (will be getting bigger and bigger)

E.g. running a task that deletes the entry after it gets invalid